File: pset6/mario/mario.py
from cs50 import get_int
height = 0

while height < 1 or height > 8 :
    height = get_int("Height: ")

spaces = height
blocks = 1
for j in range(height):
    for i in range(spaces-1):
        print(" ", end ="")

    spaces = spaces - 1
    print("#" * blocks, end="")
    print("  ", end="")
    print("#" * blocks, end="")
    blocks = blocks + 1
    print()

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

void block(int i);

int main(void)
{
    int height;
    do
    {
        height = get_int("Enter Height Of Pyramid\n");

    }
    while (height < 1 || height > 8);

    for (int i = 0; i < height; i++)
    {
        for (int s = height - 1 - i; s > 0; s--)//for loop for printing spaces on left side
        {
            printf(" ");
        }

        block(i);

        printf("  ");

        block(i);

        printf("\n");
    }
}

void block(int i)//funcion for priting the hashes depedning on the inputed height
{
    for (int b = 0; b < i + 1; b++)
    {
        printf("#");
    }
}
